packages/data-transport-layer/package.json

```diff
@@ -52,7 +52,7 @@
     "ethers": "^5.7.0",
     "express": "^4.17.1",
     "express-prom-bundle": "^6.3.6",
-    "level": "^6.0.1",
+    "level6": "npm:level@^6.0.1",
```

Comment:
@tynes this PR is marked as "protocol critical" because it touches the DTL, but this is the only change being made. I had to use this alias because:
1. We updated `smock` to 2.3.0 which now imports some new hardhat package.
2. The new hardhat package being imported itself imports level 8.
3. Our monorepo is picking up level 8 instead of level 6 for the DTL and the interface was broken so it throws a TSC error.

I resolved this issue by explicitly telling the package to use level6. This should NOT have any impact on the behavior of the DTL since it's using the exact same version of level it was using before.